modules/500-upmeter/images/upmeter/src/pkg/probe/load.go

```diff
@@ -145,7 +153,11 @@ func (l *Loader) collectConfigs() []runnerConfig {
 	l.configs = append(l.configs, initMonitoringAndAutoscaling(l.access, l.nodeLister, l.preflight)...)
 	l.configs = append(l.configs, initExtensions(l.access, l.preflight)...)
 	l.configs = append(l.configs, initLoadBalancing(l.access, l.preflight)...)
-	l.configs = append(l.configs, initDeckhouse(l.access, l.preflight, l.logger)...)
+	l.configs = append(l.configs, initDeckhouse(l.access, l.preflight, probeConfig{
```

Comment:
Why don't you just pass `l.dynamic.Converge`? `TaskGrowthThreshold` field is not used anyway, and probeConfig just duplicates the type.

Reply:
because of cyclic import